smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/ServiceGenerator.kt

```diff
@@ -86,6 +86,10 @@ class ServiceGenerator(private val ctx: RenderingContext<ServiceShape>) {
             }
             .closeBlock("}")
             .write("")
+
+        operations.forEach { op ->
```

Comment:
Why `op` instead of implicit `it`?

Reply:
Oh because I copied/pasted from the similar block up on L83. I'll simplify it.